deploifai/project/create.py

```diff
@@ -104,7 +104,12 @@ def create(context: DeploifaiContextObj, name: str, workspace):
         click.echo("Could not fetch cloud profiles. Please try again.")
         return
 
-    # TODO: prompt user if no existing cloud profiles exist
+    if not cloud_profiles:
+        click.echo(click.style("No cloud profiles found. To create a cloud profile: ", fg="red") +
+                   click.style("deploifai cloud-profile create", fg="blue")
+                   )
```

Comment:
Probably a bit too excessive with the colors here. Maybe just echo the whole line in yellow.

Reply:
sure np